cfme/common/__init__.py

```diff
@@ -79,14 +84,51 @@ def add_tag(self, tag, single_value=False):
         self.load_details(refresh=True)
         mixins.add_tag(tag, single_value=single_value, navigate=True)
 
+    def add_tags(self, tags):
+        """Add list of tags
+        tags: List of `Tag` namedtuple
+        """
+        for _tag in tags:
+            self.add_tag(tag=[_tag.category, _tag.tag_name], single_value=_tag.single_value)
+
     def remove_tag(self, tag):
         self.load_details(refresh=True)
         mixins.remove_tag(tag)
 
+    def remove_tags(self, tags):
+        """Remove list of tags
+        tags: List of `Tag` namedtuple
+        """
+        for _tag in tags:
```

Comment:
@mfalesni `_tag` used in internal hence if we put `_` may be `private` scope?
I will change this.